inc/html.class.php

```diff
@@ -2312,12 +2289,12 @@ static function getMassiveActionCheckBox($itemtype, $id, array $options = []) {
     *
     * @since 0.84
     *
-    * @param $itemtype             Massive action itemtype
-    * @param $id                   ID of the item
-    * @param $options      array
+    * @param string  $itemtype Massive action itemtype
+    * @param integer $id       ID of the item
+    * @param array   $options
     *
-    * @return show checkbox
-   **/
+    * @return string
```

Comment:
It returns void.

Reply:
echo returns a string so at the end this function is returning a string, isn't it?